### PR TITLE
New installation script and configuration options.

### PR DESCRIPTION
**Pull request by lm41**
Added a log generating script.
Merged configure_goxlr.sh and configure_goxlr_mini.sh into one run_goxlr.sh

### CODE REVIEW

**Comment by VirtualJunky** [comment]
I think on line 85 we should include `set_config cmode "false"` just in case some runs the install again there is the potential for it to get trapped as true. Not sure it would ever be an issue but good safety to have, that way it will always try default first then if it fails switch to compatibility mode.

**Comment by lm41** [comment]
Closes #17

**Comment by VirtualJunky** [comment]
Partially fixes #15 also fixes both issues mentioned in #23

**Comment by lm41** [comment]
please remove line 10 before merging into main.

**Review by VirtualJunky — COMMENTED** [review]
Works for me on Pop!_OS. Think the changes are good and opens up a lot of options in the future.

File: run_goxlr.sh [hunk 1]
```diff
@@ -61,4 +77,15 @@ jack_connect system:capture_4 GoXLR_Source_Chat:front-right
 # realtime-scheduling = yes
 # exit-idle-time = -1
 
+#Set profile of GoXLR to Multichannel Duplex. Might need another line for Mini?
+pactl set-card-profile alsa_card.usb-TC-Helicon_GoXLR-00 output:multichannel-output+input:multichannel-input
+
+#Set default output device
+#Find current index name from config name, trim unneeded characters, set default
+found=$(pacmd list-sink | grep -E "$ouput|name:" | grep -B 1 $ouput | sed '/'$ouput'/d' | sed 's/[ 	<>]//g' | sed 's/name://g')
```
Comment (T-Grave) [inline run_goxlr.sh:85]: type: should be `list-sinks` instead

**Review by T-Grave — APPROVED** [review]

**Comment by T-Grave** [comment]
On install it failed finding / setting the default devices, it just showed an empty list and a bunch of errors. After restart everything works except for default selection

**Pull request merged**